docs/graph_explorer.html

```diff
@@ -382,7 +384,13 @@
 
         `.trim();
 
-
+        const loadMachineConfig = defaultConfig =>{
+          const urlArg                = new URLSearchParams(window.location.search);
+          const compressed            = urlArg.get('s');
+          const sharedMachineConfig   = compressed ? LZString.decompressFromEncodedURIComponent(compressed) : null;
+          const machineConfig         = sharedMachineConfig || defaultConfig;
+          lconfig                     = machineConfig;
+        }
 
         editor.innerHTML = l_config;
```

Comment:
then here you could say `editor.innerHTML = loadMachineConfig(l_config)` here and you'd be done